src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/custom.py

```diff
@@ -58,7 +57,8 @@ def create_storage_account(resource_group_name, account_name, sku, location=None
         tags=tags,
         custom_domain=CustomDomain(custom_domain, None) if custom_domain else None,
         encryption=encryption,
-        access_tier=AccessTier(access_tier) if access_tier else None)
+        access_tier=AccessTier(access_tier) if access_tier else None,
+        enable_https_traffic_only=https_only)
```

Comment:
This works against latest profile only. Azure Stack profile uses another custom command, because of the `--account-type` breaking change.